src/effects/native/balanceeffect.cpp

```diff
@@ -1,7 +1,11 @@
 #include "balanceeffect.h"
 
-namespace {
+#include "util/defs.h"
 
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
Step by step: 
* All values on default 
* Turn right gain slowly to zero 
* Turn bypass slowly to 2200 Hz
* Result: Signal is back to fully dry